### PR TITLE
TASK 7: implement post comment by article id

### PR DESCRIPTION
**Pull request by YShamm**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -97,3 +97,52 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
...
+describe("POST /api/articles/:article_id/comments; adds a comment for an article by article id", () => {
+  test("posts a comment w/ a username and body", () => {
+    const newComment = { username: "rogersop", body: "great article!" }; //in my model do need to change username to author?
+
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        console.log(response, "response in TEST");
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:110]: remove console.logs before committing

File: __tests__/app.test.js [hunk 2]
```diff
@@ -97,3 +97,52 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
...
+        // expect(response.body.comment.author).toBe("rogersop");
+        expect(response.body.comment).toHaveProperty("author", "rogersop");
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:112]: .toMatchObject or .objectContaining are useful here to be thorough and check both keys and values

**Review by Beth-McCormack — COMMENTED** [review]
Nice work! A few test cases to consider

- ignores unnecessary properties
- missing required field(s), e.g. no username or body properties
- username does not exist

**Review by Poonam-raj — APPROVED** [review]
Great work taking on Beth's suggestions, Yahya, you're free to merge this in.

I would highly recommend you tidy your code of any console logs and any comments now that it's approved before merging it though.

**Pull request merged**